server/src/routes/auth.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const { signup } = require("../controllers/auth");
+const { validateSignupRequest, isRequestValidated } = require("../validators/auth");
+const router = express.Router();
+
+router.post('/signup',validateSignupRequest, isRequestValidated, signup);
+
+
+
+module.exports = router;
```

Comment:
While importing the signup don't import it's default name import the controller as specified in the `example.route.js`. Import it as `authController` and while assigning write `authController.signup` not `signup` as that's ambiguous.

And rename the file to `auth.route.js`